django_afip/templates/receipts/code_6.html

```diff
@@ -9,9 +9,10 @@
     <link rel="stylesheet" href="{% static "receipts/receipt.css" %}">
   </head>
   <body>
+    {% for page,value in entries.items %}
```

Comment:
`value` is not a very descriptive variable name:

```suggestion
    {% for page, page_entries in entries.items %}
```